api/app/models/ingestion_message.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+# A message stored during resource ingestion to be later returned to the client
+class IngestionMessage < ApplicationRecord
+  belongs_to :ingestion
```

Comment:
This should have `validates :kind, :payload, presence: true` to reflect the DB constraints.